crowdsourcing/models.py

```diff
@@ -298,6 +298,12 @@ class Task(models.Model):
 class TaskWorker(models.Model):
     task = models.ForeignKey(Task, related_name='task_workers')
     worker = models.ForeignKey(Worker)
+    statuses = ((1, 'In Progress'),
```

Comment:
why are statuses needed here? basically the results have these statuses rather than the workers